tests/packages/test_locker.py

```diff
@@ -193,7 +199,7 @@ def test_locker_properly_loads_nested_extras(locker: Locker):
 python-versions = "*"
 
 [package.dependencies]
-c = {version = "^1.0", optional = true}
+c = {{version = "^1.0", optional = true}}
```

Comment:
What's going on here with these extra braces?

Reply:
This is how you escape `f` string formatting:
```
In [1]: my_str = "{hello world} {bar}"

In [2]: bar = "baz"

In [3]: my_str = f"{hello world} {bar}"
  Input In [3]
    my_str = f"{hello world} {bar}"
           ^
SyntaxError: f-string: invalid syntax


In [4]: my_str = f"{{hello world}} {bar}"

In [5]: print(my_str)
{hello world} baz
```